plugin.yaml

```diff
@@ -252,6 +252,8 @@ node_types:
           inputs:
             workflow_id:
               default: install
+            timeout:
+              default: 120
```

Comment:
The current default timeout is `900` (see `constants.EXECUTION_TIMEOUT`). Why are we making it `120`?

Reply:
@isaac-s Yeah I know it is 900 but I just add it here to make sure  this update to ``plugin.yaml make`` sense and does not confusing the end user and if we really need to add anything else.
please let me know what do you think